recaf-ui/src/main/java/software/coley/recaf/services/info/summary/builtin/MinecraftModSummarizer.java

```diff
@@ -0,0 +1,225 @@
+ * @Author Canrad
+ */
+@ApplicationScoped
+public class MinecraftModSummarizer implements ResourceSummarizer {
+    private final TextProviderService textService;
+    private final IconProviderService iconService;
+    private final Actions actions;
+
+    @Inject
+    public MinecraftModSummarizer(@Nonnull TextProviderService textService,
+                                  @Nonnull IconProviderService iconService,
+                                  @Nonnull Actions actions) {
+        this.textService = textService;
+        this.iconService = iconService;
+        this.actions = actions;
+    }
+
+    @Override
+    public boolean summarize(@Nonnull Workspace workspace,
+                             @Nonnull WorkspaceResource resource,
+                             @Nonnull SummaryConsumer consumer) {
+        Batch batch = FxThreadUtil.batch();
+        boolean foundAny = false;
+
+        // Add title
+        batch.add(() -> {
+            Label title = new BoundLabel(Lang.getBinding("service.analysis.minecraft-mod-info"));
+            title.getStyleClass().addAll(Styles.TITLE_4);
+            consumer.appendSummary(title);
+        });
+
+        // 1. Try to find Fabric mod information
+        FileInfo fabricFileInfo = resource.getFileBundle().get("fabric.mod.json");
+        if (fabricFileInfo != null) {
+            foundAny = true;
+            String mcVersion = "";
+            List<String> mainClasses = new ArrayList<>();
+
+            try {
+                String jsonText = fabricFileInfo.asTextFile().getText();
+                JsonObject json = JsonParser.parseString(jsonText).getAsJsonObject();
+                if (json.has("entrypoints")) {
+                    JsonObject entrypoints = json.getAsJsonObject("entrypoints");
+                    if (entrypoints.has("main")) {
+                        JsonArray mainArray = entrypoints.getAsJsonArray("main");
+                        for (int i = 0; i < mainArray.size(); i++) {
+                            String mainClass = mainArray.get(i).getAsString();
+                            mainClasses.add(mainClass);
+                        }
+                    }
+                }
+
+                if (json.has("depends")) {
+                    JsonObject depends = json.getAsJsonObject("depends");
+                    if (depends.has("minecraft")) {
+                        mcVersion = depends.get("minecraft").getAsString();
+                    }
+                }
+            } catch (Exception e) {
+                // Ignore JSON parsing errors
+            }
+
+            String finalMcVersion = mcVersion;
+            batch.add(() -> {
+                Label title = new BoundLabel(Lang.getBinding("service.analysis.is-fabric-mod"));
+                consumer.appendSummary(title);
+
+                if (!finalMcVersion.isEmpty()) {
+                    consumer.appendSummary(new Label(Lang.getBinding("service.analysis.minecraft-version").get() + " " + finalMcVersion));
```

Comment:
You can use `String.format` templates in bindings, see `menu.mappings.export.unsupported` for example:
```java
// menu.mappings.export.unsupported=%s (Unsupported)
Lang.formatLiterals("menu.mappings.export.unsupported", formatName)
```